pkg/preference/preference.go

```diff
@@ -294,6 +310,16 @@ func (c *PreferenceInfo) GetExperimental() bool {
 	return *c.OdoSettings.Experimental
 }
 
+// GetPushTarget returns the value of PushTarget from config
```

Comment:
`from preference`

Reply:
Good catch! I've updated the other functions from preferences that were referencing config as well.